platform/platform-api/src/main/java/org/codice/ddf/platform/api/Describable.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.api;
+
+/**
+ * Describable is used to capture a basic description.
+ */
+public interface Describable {
```

Comment:
Can we check to see if any of the implementations we have of the old interface ever return `null` for any of those methods? If not, we may want to indicate (`@NotNull`?) which ones can return `null` and which ones never will. We should also mention which ones could return an empty string and which ones won't (`@Min()`?).